### DISCUSSION TITLE
Faulty forwarded port 400 (Bad Request)

### DISCUSSION THREAD

**Discussion by tachibana-shin** [community/community#17928] [Codespaces]
Hi,

I've been granted the right to test codepaces personally. It's great, but there's a problem with port forwarding.
I successfully redirect ports but when opening redirected URLs, cannot delegate and cause `400 (Bad Request)' error

I try changed repo to public but not working

![image](https://user-images.githubusercontent.com/45375496/172034278-76ee2c0b-7f0d-445c-872a-ab46c6c706a6.png)
![image](https://user-images.githubusercontent.com/45375496/172034292-045502c2-cbe2-470e-b06d-864ba7eaae16.png)

**Comment by yuy678** [comment]
Ping router adres

**Reply by tachibana-shin** [reply]
> 
What do you mean?
